test/CMakeLists.txt

```diff
@@ -1,9 +1,13 @@
 file(GLOB_RECURSE sources "*.cpp")
-list(REMOVE_ITEM sources "${CMAKE_CURRENT_SOURCE_DIR}/fuzzer.cpp")
+file(GLOB_RECURSE toolsources "tools/*.cpp")
+list(REMOVE_ITEM sources ${toolsources})
 file(GLOB_RECURSE headers "*.h")
 
 add_executable(soltest ${sources} ${headers})
 target_link_libraries(soltest PRIVATE libsolc solidity lll evmasm devcore ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES})
 
-add_executable(solfuzzer fuzzer.cpp)
+add_executable(solfuzzer tools/fuzzer.cpp)
 target_link_libraries(solfuzzer PRIVATE libsolc evmasm ${Boost_PROGRAM_OPTIONS_LIBRARIES})
+
+add_executable(syntaxTestTool tools/syntaxTestTool.cpp TestHelper.cpp libsolidity/SyntaxTestParser.cpp libsolidity/AnalysisFramework.cpp)
```

Comment:
Should I add ``EXCLUDE_FROM_ALL`` here? The ``syntaxTestTool`` won't be of much use for most users - in contrast to ``solfuzzer`` it won't even be used for running any tests, so maybe it makes more sense to require to explicitly build it with ``make syntaxTestTool``?

Reply:
I think we should always build everything, to see when it breaks. Most users don't actually build from source.